### ISSUE TITLE
return only one response, UNKNOWN_COMMAND, to the command longer than LINE_BUF_SIZE

### ISSUE THREAD

**Issue by etanol** [beanstalkd/beanstalkd#141]
So far, I've seen two situations where the server and the client can become out of sync:
1. The client sends a command longer than LINE_BUF_SIZE.
2. The client sends a syntactically wrong `put` command with a job payload.

There maybe more.  Right now the server reports the error as soon as it detects it and keeps going.  But the connection stream is then out of sync for an undefined number of invalid command cycles.  Wouldn't be more appropriate just to punish the client and disconnect it?

**Comment by ysmolsky** [comment]
I do not think that this should be fixed on the server side except for this weird glitch. I gueess things have changed since 2012. So if the clients sends too long command, then two responses will be sent to the client. In the case of wrong payload, it's a job of a client to make sure that it does not sent broken data, I do not see how server can fix that.

```
% nc 127.0.0.1 11000 -c
kfjlksdfjklasjdflkajsd;flkjasd;lkfjaslkdfjalksdjf;laksjdflkasjdflkjasdlkfjaslkdfjl;askdjflkasjdflkasjdflkjasdl;fkjasl;kdfjals;kdjflaksjdfl;kasjdflkasjdfl;kajsdlfkjasldkfjaslkdfjaslkdfjhasjkdhfkjasdhfkjashdfkjashdkfjhfaslkdfjalksdjflkasjdfkljasdf
BAD_FORMAT
UNKNOWN_COMMAND
putqwjeklqjwkejqwkejklqwjelkqjwlkejqwklejqklwjelkqjwkljeklqjwkejqklwjeklqjwekljqwkeljqklwjeklqwjekljqwekjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj
BAD_FORMAT
UNKNOWN_COMMAND
delete 1
NOT_FOUND
put a a a a
BAD_FORMAT
ekq;lwke;lqkwe;lqkw;ekqlwke;qej;lkwqjerkljwqelkrjlwqkjer;lkwqjerlkjwqelkrjqwlkejrlkwqjerlkwjelrkjqwlkejr;lkqwjerlkwqjer
UNKNOWN_COMMAND
```

**Comment by ysmolsky** [comment]
Fixed by #562
